packages/@ember/debug/container-debug-adapter.ts

```diff
@@ -88,9 +86,9 @@ export default class ContainerDebugAdapter extends EmberObject {
     @return {Array} An array of strings.
     @public
   */
-  catalogEntriesByType(type: string): NativeArray<string> {
-    let namespaces = emberA(Namespace.NAMESPACES);
-    let types = emberA<string>();
+  catalogEntriesByType(type: string): string[] {
+    let namespaces = Namespace.NAMESPACES;
+    let types: string[] = [];
```

Comment:
Hmm. Documentation is very apt to be incorrect in this space, though; folks who wrote the docs likely were not thinking about the differences (because the differences are subtle!). Let's avoid changing the behavior here—we can always do so in a later PR, but keeping types changes separate from behavior changes will help us avoid snafus.